### DISCUSSION TITLE
ReactDOM version

### DISCUSSION THREAD

**Discussion by cigzigwon** [code4recovery/12-step-meeting-list#514] [Ideas]
Since I don't currently use Wordpress I would like to create a React implementation of this plugin. You only really need to hook the compiled version to an ephemeral JSON string and it should work. And then in theory you could still hook it in WP on the fly usage anyhow.

Our datasources are going to be read from GraphQL here in District 28 so when it spews it JSON the new React impl will handle it. I personally stopped using jQuery as well.

So more than likely I will just share the repo and you can choose to fork it since it will be using just Webpack 4 and ES6 w/React

**Comment by tech2serve** [comment]
Please check out and contribute to our [https://github.com/code4recovery/tsml-ui](TSMLui) project. This is a React-based meeting finder. :-)

I have thought about a React-based backend/meeting management tool, but our current focus is on making TSMLui work with spreadsheets for tracking and managing meeting information.

**Reply by cigzigwon** [reply]
@tech2serve I did not see that! I guess no need for it. I did find the standalone TS script based online CRACO app very interesting. I think I'm just going to run two sites using that instead for our meeting listing. Then we can have meetings.district28aa.online hosting it in Netlify. What do you think?

**Reply by tech2serve** [reply]
If you're an AA member, consider joining our [NAATW 2021](https://naatw.org) annual workshop. Code for Recovery will be presenting tomorrow, and we'll talk more about TSML UI.
